MediaBrowser.Model/Dlna/ResolutionNormalizer.cs

```diff
@@ -15,7 +15,6 @@ public class ResolutionNormalizer
                 new ResolutionConfiguration(720, 950000),
                 new ResolutionConfiguration(1280, 2500000),
                 new ResolutionConfiguration(1920, 4000000),
-                new ResolutionConfiguration(2560, 8000000),
```

Comment:
Why did you remove this?

Reply:
Currently there is no 1440p options in web client.
May be related to https://github.com/jellyfin/jellyfin-web/pull/1993

Reply:
I would like to increase it to 20Mbps for 1440p, as 8Mbps is inadequate for 1440p H264 transcoding.

Hardware encoding has a side effect on quality. We may also have 60fps videos. 

YouTube recommends 16~24Mbps for 1440p video uploading. They re-encode the video to VP9 to lower the bandwidth.